### DISCUSSION TITLE
Release process of renovate bot

### DISCUSSION THREAD

**Discussion by olira** [renovatebot/renovate#19985] [Suggest an Idea]
### Type of discussion.

I just want to chat

### Tell us more.

Hi renovate team,

While looking at the recent releases of renovate, it seems like not each push to the main branch leads to a release of renovatebot, but a rather semantic release process is used.

Given the amazing frequency of releases, even throughout the day, I'm wondering whether releases are triggered manually or whether there is some automated mechanics to cut a release.

Is there any documentation available on the release process? I've been looking through the developer documentation but didn't find any specifics.

**Comment by viceice** [comment]
this is entirely done automatically. Releases depends on semantic commit type.

**Reply by olira** [reply]
Ahh, I see that makes sense. Is there any config file I can look at to see what types were chosen to trigger a release?

**Reply by rarkins** [reply]
We use https://github.com/semantic-release/semantic-release with default settings (feat=minor release, fix/build=patch)

**Reply by olira** [reply]
Ah that's awesome and a good pointer. I was a bit confused with all this DRY_RUN config in build.yaml, but after re-visiting and walking it through, in conjunction with the hint about semantic-release and default it became clear 😅 

Thanks for all the great information.
